camera_cards

```diff
@@ -312,13 +321,45 @@ if [[ "${TAR}" == "YES" ]] ; then
     ## tar supplied input directory
     echo
     _report -g "Creating tar archive..."
+    _writelog "Tar archive process process started" "$(date +%FT%T)"
     tar -czvf "${AIPDIR}/objects/${MEDIAID}.tar.gz" -C "${CAMERA_CARD_DIR}" .
     _writelog "TAR ARCHIVE" "${MEDIAID}.tar.gz"
+    _writelog "Tar archive process process finished" "$(date +%FT%T)"
+fi
+
+# error reporting
+_report -b "Checking for errors during the camera_cards process..."
+if [[ -z $(cat "${TREE}") ]] || [[ -z $(cat "${MEDIAINFO_OUTPUT}") ]] || [[ -z $(cat "${EXIFTOOL_OUTPUT}") ]] || [[ -z $(cat "${FFPROBE_OUTPUT}") ]] ; then
+    _report -r "Metadata generation process created one or more blank files. Input files may be empty. Review metadata output at ${METADATA_OUTPUT_DIR}."
+    _writelog "POSSIBLE_ERROR_REVIEW" "Metadata generation process created one or more blank files"
+else
+    _report -g "Metadata generation process looks ok"
+fi
+if [[ "${FFMPEG_EXIT_CODE}" -gt 0 ]] || [[ "${FFMPEG_ERRORS}" -gt 0 ]] ; then
+    _report -r "FFmpeg may have encountered errors - review terminal output for more detail."
+else
+    _report -g "FFmpeg exit codes look ok"
+fi
+if [[ -s "${CONCATENATED_AUDIO_FILE}" ]] ; then
+    if [[ "${VIDEO_DURATION}" -ne "${AUDIO_DURATION}" ]] ; then
+    _report -r "Concatenated audio and video files were not the same length - review final merged file for sync at ${CONCATENATED_FILE} (video duration = ${VIDEO_DURATION}, audio duration = ${AUDIO_DURATION})"
+    _writelog "POSSIBLE_ERROR_REVIEW" "Concatenated audio and video files were not the same length"
```

Comment:
What precision is the duration comparison at?  Like one NTSC frame with 48000 audio the closest duration match would be a video frame at 1001/30000 and audio at 1602/48000.
aka
0.03336666667 (video) versus 0.033375 (audio) is the closest duration alignment you can get. The next lower 480000 audio duration is at 1601/48000 or 0.03335416667 which is even further off.

Reply:
Hmmm, good point. It's currently reporting the output of `mediainfo --Inform="Video;%Duration%"` and `mediainfo --Inform="Audio;%Duration%"`, so milliseconds. I was looking at dvpackager and it looks like it is using duration in seconds ([? hope I'm reading it right](https://github.com/mipops/dvrescue/blob/4e76bd384b8feec2e270ed8d42c87b5f2ba71dee/tools/dvpackager#L416)). Maybe better to round this to seconds?